cdap-docs/build.sh

```diff
@@ -408,8 +408,16 @@ function build_license_dependency_pdfs() {
 }
 
 function build_standalone() {
```

Comment:
I don't think most people will use this, but it isn't clear to a user that they need hydrator-plugins to build cdap using this command/function.